website/content/docs/concepts/domain-model/targets.mdx

```diff
@@ -58,56 +58,6 @@ TCP targets have the following additional attributes:
   You can configure an egress filter to enable [multi-hop](/boundary/docs/configuration/worker/pki-worker#multi-hop-workershcp-only) connections.
   If you do not configure an egress filter, then Boundary uses a single worker to connect to the controller.
```

Comment:
this section contains references to multi-hop and HCP, do you want to leave it in?

Reply:
Oops, definitely not. That link won't be good until the 14th. I removed both of these sentences. Thanks for catching this!